Frends.AzureBlobStorage.UploadBlob/Frends.AzureBlobStorage.UploadBlob/Definitions/Input.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.AzureBlobStorage.UploadBlob
+{
+    /// <summary>
+    /// Input-class for UploadBlob-task.
+    /// </summary>
+    public class Input

```

Comment:
The same comment about Input and Options vs Source and Destination - maybe some better naming for those? Because options currently looks like destination, and not just options. Maybe even bring source and destination to input then and put only optional things to the options tab somehow?

Reply:
Renamed to source/destination.